libkbfs/folder_update_prepper.go

```diff
@@ -331,6 +331,13 @@ type pathTreeNode struct {
 	mergedPath path
 }
 
+type prepFolderFlags byte
+
+const (
+	// The blocks of indirect files should be copied.
+	prepFolderCopyIndirectFileBlocks prepFolderFlags = 1 << iota
```

Comment:
Because there's no need to remember the right incantations to test for a flag, and no temptation to get "clever" with testing multiple flags, e.g.:

```
((flags & prepFolderFooOn) != 0) && ((flags & prepFolderBarOn) != 0)
```

or

```
((flags & (prepFolderFooOn | prepFolderBarOn)) != 0)
```

or

```
((flags & prepFolderFooOn) == prepFolderFooOn) && ...
```

(The second one is wrong, btw.)

It's possible to make similar mistakes with separate flags, I guess, but it seems harder to do (because all you can do is check equality).